File: CS390/number2.cs
public class number2
{
	static int fun(int k)
	{
		k += 4;

		return 3 * k - 1;
	}

	public static void Main(string[] args)
	{
		int i = 10, j = 10, sum1, sum2;
		sum1 = (i / 2) + fun(i);
		sum2 = fun(j) + (j / 2);

		System.Console.WriteLine("sum1 = " + sum1);
		System.Console.WriteLine("sum2 = " + sum2);
	}
}

File: CS390/number2.cpp
#include <iostream>

int fun(int *k); //could also do by reference like: int fun(int &k);
                 //that way, the call would be fun(i); instead of fun(&i);
int main()
{
	int i = 10, j = 10, sum1, sum2;
	sum1 = (i / 2) + fun(&i);
	sum2 = fun(&j) + (j / 2);

	std::cout << "sum1 = " << sum1 << '\n';
	std::cout << "sum2 = " << sum2 << '\n';

	return 0;
}

int fun(int *k)
{
	*k += 4;

	return 3 * (*k) - 1;
}
